### PR TITLE
Move the configuration and scrpt render from workspace to CBMC

### PR DESCRIPTION
**Pull request by xiar**
* move the infrasim.yml and vbmc.conf to workspace/etc
* move the script render from class WorkSapce to class CBMC

### CODE REVIEW

File: infrasim/model.py [hunk 1]
```diff
@@ -1658,29 +1700,34 @@ def init(self):
...
+            self.__config_file = self.__bmc['config_file']
+            if os.path.exists(self.__config_file):
+                shutil.copy(self.__config_file,
+                            os.path.join(self.get_workspace(), "etc/vbmc.conf"))
+        elif self.get_workspace() and not self._task_is_running():
+            # render template
+            self.__render_template()
+            self.write_bmc_config(os.path.join(self.get_workspace(), "etc/vbmc.conf"))
+        elif os.path.exists(os.path.join(self.get_workspace(), "etc/vbmc.conf")):
+            self.__config_file = os.path.join(self.get_workspace(), "etc/vbmc.conf")
```
Comment (XiaowenJiang) [inline infrasim/model.py]: 666

**Comment by xiar** [comment]
retest this please

**Comment by turtle-fly** [comment]
Glad to see the change! Go through test then we can merge~

**Comment by XiaowenJiang** [comment]
retest this please
I'm not sure why the job randomly fails after clone.

**Comment by XiaowenJiang** [comment]
retest this please

**Comment by XiaowenJiang** [comment]
retest this please

**Comment by XiaowenJiang** [comment]
retest this please

**Comment by XiaowenJiang** [comment]
retest this please
Hope I got the test config recovered....

**Comment by xiar** [comment]
retest this please

**Comment by xiar** [comment]
retest this please

**Pull request merged**